bin/sras

```diff
@@ -50,6 +50,22 @@ const scanConfig = parseSRASConfig(config);
 
 console.log(`Scanning assets at [path = ${program.path}]`);
 
+const scanJs = async (filePath, scanRules) => {
+  try {
+    const stat = await fsStat(filePath);
+    const maxSize = scanRules.maxSize || Infinity;
+    if (maxSize > 0 && stat.size > maxSize) {
+      scanReports.push(buildReportMsg(
+        'JS file size is larger than the recommended file size',
+        `[recommended = ${formatHRTFileSize(maxSize)}],`,
+        `[${filePath} = ${formatHRTFileSize(stat.size)}]`,
```

Comment:
I would suggest only reporting the file name instead of the full file path. This will likely result in the boating the report, making it less readable. This will also keep it consistent with the other checks' reports.

Alternatively, you could report the parent directory along with the file name.

Reply:
Good point, I'll fix that.